torch/_inductor/fx_passes/quantization.py

```diff
@@ -117,6 +117,31 @@ def _may_generate_pattern_with_dtype_convert(pattern, dtype=Arg(), dtype_convert
     Arg(),  # algorithm
 )
 
+
+def get_dequantize(users=1):
```

Comment:
Can we extend `dequantize_qconv_pt2e_pattern` to avoid duplicate code?

Reply:
Thank you for your feedback! I made a change to utilze get_dequantize for where using `dequantize_qconv_pt2e_pattern` to avoid the duplicate.

Reply:
Please give it a meaningful name here, e.g., `get_dequantize_qconv_pt2e_pattern`.